hm_4.py

```diff
@@ -0,0 +1,12 @@
+"""
+Write a Python program to get the current time in Python.
+"""
+
+import time
+
+
+def return_time():
+    return time.ctime()
```

Comment:
Do you know other alternatives with `datetime` module?

Reply:
I know

Reply:
Which one?

Reply:
So, why do you decide to use `time.ctime()`?

Reply:
because time.ctime () shows the same result as datetime.now ()

Reply:
But it's not popular solution.

Reply:
I understood, changed